salt/kafka-tool/init.sls

```diff
@@ -0,0 +1,75 @@
+#read all pillar data
+
+{% set install_dir = pillar['pnda']['homedir'] %}
+{% set packages_server = pillar['packages_server']['base_uri'] %}
+{% set release_directory = salt['pillar.get']('kafkatool:release_dir', '/opt/pnda') %}
+{% set release_version = salt['pillar.get']('kafkatool:release_version', 'v0.2.0')  %}
+{% set release_filename = 'kafka-tool-' + release_version + '.tar.gz' %}
+
+{% set p  = salt['pillar.get']('kafka', {}) %}
+{% set local_kafka_path = p.get('prefix', '/opt/pnda/kafka') %}
+{% set local_kafkat_log_path = salt['pillar.get']('kafkatool:log_dir', '/var/log/pnda/kafkatool') %}
```

Comment:
if used on kafkat configuration file, should be kafka log dir in order to clean the indexes (as defined in the kafka pillar, which is /var/kafka-logs)